src/test/java/com/wikia/webdriver/common/remote/operations/CreateCategory.java

```diff
@@ -29,11 +29,16 @@
         .put("siteId", context.getSiteId())
         .build());
 
-    final String response = remoteOperation.execute(buildUrl(context), jsonObject);
+    String response = null;
+    try {
+      response = remoteOperation.execute(buildUrl(context), jsonObject);
+    } catch(RemoteException e) {
+      PageObjectLogging.logError("error: ", e);
+    }
 
     DocumentContext json = JsonPath.parse(response);
```

Comment:
If the remoteOperation.execute fails you will get a null pointer exception here.
I would suggest moving the response evaluation to another method and re-throw the RemoteException.